### ISSUE TITLE
ProofGeneral menu is empty

### ISSUE THREAD

**Issue by JasonGross** [psteckler/ProofGeneral#60]
![image](https://cloud.githubusercontent.com/assets/396076/20687783/d8333478-b58b-11e6-83f9-c47f32275262.png)

**Comment by JasonGross** [comment]
The `Coq` menu is also empty.  This is emacs-26.0.50-snapshot

**Comment by psteckler** [comment]
OK, I suspect this is emacs-version-related.

When you pull down those menus, do you see anything untoward in the *Messages* buffer?

**Comment by psteckler** [comment]
Also, do the menus in vanilla PG work OK with your emacs version?

**Comment by JasonGross** [comment]
Indeed, both PG-xml and vanilla PG menus work fine in emacs 24, and are empty in emacs 26.  Note that in Emacs 24, menus show up at the top of the (unmaximized) window, while in emacs 26, menus show up on the top of the system toolbar.

**Comment by JasonGross** [comment]
This command-line warning may be relevant:
```
(emacs-26-0-50-snapshot:4462): Gtk-WARNING **: Attempting to add a widget with type GtkBox to a GtkMenuItem, but as a GtkBin subclass a GtkMenuItem can only contain one widget at a time; it already contains a widget of type GtkAccelLabel
```

**Comment by psteckler** [comment]
The latest Emacs release appears to be 25.1, so I don't think it's a high priority to fix this. It might be an Emacs bug (though maybe it's simply that the menuing setup has changed). 

Can we leave this open, and if Emacs 26 is released, look at it again?

**Comment by psteckler** [comment]
FWIW, I tried running PG/xml with "GNU Emacs 26.0.50.1 (x86_64-pc-linux-gnu, GTK+ Version 2.24.30)", and the menus appear OK.

**Comment by ejgallego** [comment]
This is likely an Ubuntu regression, are you using Ubuntu @psteckler ?

**Comment by psteckler** [comment]
I'm using Linux Mint 18, which is a fork of Ubuntu.

**Comment by ejgallego** [comment]
Mint with Cinammon? I can't bet of course, but I'd say this is a window-manager (Unity I am looking at you) specific issue.

**Comment by psteckler** [comment]
Yes, Mint w/ Cinnamon.

**Comment by ejgallego** [comment]
You can try logging with Unity and see what happens; it is very likely that this is a Unity/emacs problem and the bug should be moved to the Unity/Emacs bug tracker.

**Comment by psteckler** [comment]
I started to try to install Unity on Mint, but the number of packages required is huge, which scared me off.
In any case, it's not a PG bug.

**Comment by psteckler** [comment]
FWIW, Ubuntu just announced they are switching to Gnome and abandoning Unity.

**Comment by ejgallego** [comment]
OMG!
